Fabric.Identity.API/Persistence/CouchDb/Stores/CouchDbResourceStore.cs

```diff
@@ -41,7 +41,7 @@ public Task<ApiResource> FindApiResourceAsync(string name)
             return DocumentDbService.GetDocument<ApiResource>(name);
```

Comment:
Can't we remove anything dealing with couchdb, since I don't believe we are using it as a store?

Reply:
It looks like a lot of this code is part of the previous pull request for aspnetcore 2.2 upgrade. I made comments in that pull request dealing with the upgrade.